README.md

```diff
@@ -254,6 +254,7 @@
 - 📚 [tails-ui](https://github.com/knipferrc/tails-ui) - React UI library using Tailwind CSS.
 - 📚 [tails](https://github.com/thedevdojo/tails) - Hand-crafted templates and components using Tailwind CSS.
 - 📚 [Svelte Headless UI](https://github.com/rgossiaux/svelte-headlessui) - Unofficial Svelte port of Headless UI.
+- 📚 [Headless UI Float](https://headlessui-float.vercel.app) - Easy use Headless UI with Floating UI to position floating elements.
```

Comment:
```suggestion
- 📚 [Headless UI Float](https://headlessui-float.vercel.app) - Floating UI integration for Headless UI.
```

How do you feel about this description?

Also, can you add a link to the GitHub repository in the docs, or update this one to point to the repository instead of the docs?

Reply:
Hmm... I think this description is not accurate since it's not just integrating Floating UI, but also has other features.

If you think my original description is not good enough, how about this: "Easy floating the Headless UI components."?

---

The GitHub repository link (https://github.com/ycs77/headlessui-float) is now in the docs:

![messageImage_1652755903567](https://user-images.githubusercontent.com/38133356/168719800-19abcc56-3398-478e-867e-4ad2f022b38a.jpg)

Reply:
I don't think it's an issue that it has more features than just being an integration tbh. But I suggested a description change because your current description is not proper English. 

I think you meant to say "Easily use Headless UI with Floating UI to position floating elements"? Even with that, there's repetitions and it doesn't really tell me what the library does. 

Sorry for being picky, I just want the list entries not to look "cheap" :p 

As for the docs, I don't see the link 🤔 

![image](https://user-images.githubusercontent.com/16060559/168797638-400e4950-58c3-4e10-9642-b75800422b2d.png)

Reply:
Ah, never mind, it was my ad blocker that blocked the social links. My bad